### PR TITLE
support container prune

### PR DESCRIPTION
**Pull request by junnplus**
Signed-off-by: Ye Sijun <junnplus@gmail.com>

Close: https://github.com/containerd/nerdctl/pull/806 https://github.com/containerd/nerdctl/pull/997

### CODE REVIEW

File: cmd/nerdctl/container_prune.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		return err
+	}
+
+	if !force {
+		var confirm string
+		fmt.Fprintf(cmd.OutOrStdout(), "%s", "WARNING! This will remove all stopped containers.\nAre you sure you want to continue? [y/N] ")
+		fmt.Fscanf(cmd.InOrStdin(), "%s", &confirm)
+
+		if strings.ToLower(confirm) != "y" {
+			return nil
```
Comment (AkihiroSuda) [inline cmd/nerdctl/container_prune.go:59]: error when the answer is not y nor N ?

File: cmd/nerdctl/rm.go [hunk 2]
```diff
@@ -50,18 +50,17 @@ func newRmCommand() *cobra.Command {
 	return rmCommand
 }
 
+type statusError struct {
```
Comment (AkihiroSuda) [inline cmd/nerdctl/rm.go:53]: Needs description

File: cmd/nerdctl/container_prune.go (hunk 1 above)
Comment (junnplus) [reply cmd/nerdctl/container_prune.go:59]: keep the same with docker.

File: cmd/nerdctl/rm.go (hunk 2 above)
Comment (junnplus) [reply cmd/nerdctl/rm.go:54]: done.

**Comment by AkihiroSuda** [comment]
Windows CI is failing

```
=== RUN   TestPruneContainer
    container_prune_test.go:41: assertion failed: res.ExitCode is 0
--- FAIL: TestPruneContainer (113.89s)
```

**Comment by junnplus** [comment]
@AkihiroSuda Fixed, only test Linux first

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**